Cabal-syntax/src/Distribution/PackageDescription/FieldGrammar.hs

```diff
@@ -268,6 +268,7 @@ data TestSuiteStanza = TestSuiteStanza
     , _testStanzaMainIs     :: Maybe FilePath
     , _testStanzaTestModule :: Maybe ModuleName
     , _testStanzaBuildInfo  :: BuildInfo
+    , _testStanzaCodeGenerators :: [String]
```

Comment:
`String` isn't a good name for a type as it does not reflect its intended semantics.  It is also badly searchable across the code base.
```haskell
type CodeGenerator = String
type CodeGenerators = [CodeGenerator]
```
and then use these type synonyms.  
Extra points for a `newtype`!